azure-pipelines-gcp-integration.yml

```diff
@@ -26,7 +26,7 @@ stages:
             displayName: 'Update pip'
 
           - script: |
-              pip install -r requirements-dev.txt --constraint constraints-dev.txt
+              pip install --requirement requirements-dev.txt --constraint constraints-dev.txt
```

Comment:
Let's opt for the more verbose option for better documentation in our scripts

Reply:
In general, I agree. I slightly prefer the shorthand here though.
Not a blocker.